firmware/steering/src/steering_control.cpp

```diff
@@ -43,9 +44,21 @@ void check_for_operator_override( void )
 
         read_torque_sensor( &torque );
 
-        float diff = (float)torque.high - (float)torque.low;
+        uint16_t unfiltered_diff = abs((int)torque.high - (int)torque.low);
```

Comment:
This section has a few style inconsistences.

Lines 47, 51, and 61 have parentheses with missing spaces inside of them.

Lines 57-59 should be indented once more.

Reply:
Updated with fb5bb4e